components/table/stories/template.js

```diff
@@ -81,7 +84,10 @@ export const TableRowItem = ({
 		${when(showCheckbox && !isSectionHeader, () => html`
 			<${cellTag}
 				role="gridcell"
-				class="spectrum-Table-cell spectrum-Table-checkboxCell"
+				class=${classMap({
+					[`${rootClass}-cell`]: true,
+					[`${rootClass}-checkboxCell`]: true,
+				})}
```

Comment:
Maybe I didn't need to do this, but it felt out of place. There aren't very many instances where I remembered us just flat out doing the class names without the `classMap()`.

Reply:
I love this!  It's a great addition.